app/utils/press-release-snapshot.js

```diff
@@ -63,6 +77,16 @@ export default class PressReleaseSnapshot {
       await publicationEvent.save();
     }
 
+    //deletes any files from the database which are in a deleted state but not actually deleted
+    const attachments = this.attachments;
+    if (attachments.length) {
```

Comment:
Deze check is overbodig. `.forEach()` (of `.map()`) werken ook op een lege array